src/main/resources/templates/home.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Strona główna</title>
+    <link rel="stylesheet" th:href="@{/styles/main.css}">
+</head>
+<body>
+<div class="logo-container">
+    <img src="../static/img/weather.jpg" th:src="@{/img/weather.jpg}"
+         alt="Twoja pogoda" class="pic"/>
+    <h1>Sprawdź pogodę dla Twojego miasta!</h1>
+</div>
+
+<form class="login-form" th:action="@{/weather}" method="post">
```

Comment:
to lepiej zrobić przez GET, żeby można później skopiować url, albo dodać sobie do ulubionych